src/main/kotlin/no/nav/familie/ef/sak/amelding/InntektResponse.kt

```diff
@@ -17,16 +19,35 @@ data class InntektResponse(
             .sumOf { it.beløp }
             .toInt()
 
-    fun førsteMånedOgInntektMed10ProsentØkning(minimumsdato: YearMonth) =
-        inntektsmånederUtenEfYtelser(minimumsdato)
-            .associate { it.måned to it.totalInntekt() }
-            .entries
-            .zipWithNext()
-            .firstOrNull { it.first.value * 1.1 <= it.second.value }
-            ?.second
-            ?.toPair()
+    fun totalInntektForÅrMåned(årMåned: YearMonth): Int =
+        inntektsmånederUtenEfYtelser(årMåned)
+            .filter { it.måned.isEqualOrAfter(årMåned) && it.måned.isEqualOrBefore(årMåned) }
```

Comment:
isEqualOrAfter -> isEqual?

Reply:
Test, eller kommentar på - flere inntektsmåneder kan gjelde samme mnd -> defror behov for som + flatmap?